docs/assets/templates/notebooks/template-small.dockstore.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.2
+notebooks:
+  - format: <JUPYTER>
+    language: <PYTHON | R | JAVASCRIPT | SCALA | JULIA | OTHER>
+    path: <String>
+    kernel: <String>
+    otherFiles: <String Array>
+    testParameterFiles: <String Array>
+    readMePath: <String>
+    authors:
```

Comment:
Hmmm, the others look better, but looks like this one lost orcid altogether

Reply:
The `template-small` templates don't have orcid. Should I add them?

Reply:
Ah ok. I guess the idea with the small ones was to provide a more minimal example. 
Technically, the one with orcid is shorter, but if they don't have an orcid account, then the process as a whole may be longer. Maybe just leave it for now then and we can harvest data long term to see how people are reacting